src/components/backend-ai-resource-monitor.ts

```diff
@@ -643,25 +649,41 @@ export default class BackendAiResourceMonitor extends BackendAIPage {
                   class="start"
                   progress="${this.used_resource_group_slot_percent.cpu /
                   100.0}"
-                  description="${this.used_resource_group_slot.cpu}/${this
-                    .total_resource_group_slot.cpu}"
+                  description="${this._prefixFormat(
+                    this.used_resource_group_slot.cpu,
+                    1,
+                  )} / ${this._prefixFormat(
+                    this.total_resource_group_slot.cpu,
+                    1,
+                  )} ${_t('summary.Cores')}"
```

Comment:
I'm just curious. Why did you add a translation only for this?

Reply:
I was going to add it because the webui doesn't provide a translation for the units, but then I realized that it's better to unify the units in English. I think it's better to remove it.